expense/expense.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Schema.Types.ObjectId;
+
+const Expense = new mongoose.Schema({
+    amount: Number,
+    description: String,
+    budget: [{
```

Comment:
Since Budget is a one-to-many relationship with Expenses, the `budget` property here _shouldn't_ be an array. Instead, it's just a single ref.